File: src/main/java/com/cyt/student/Student.java
package com.cyt.student;

public class Student {
    String id;
    String name;
    int english;
    int math;

    public Student(String name, int english, int math) {
        this.name = name;
        this.english = english;
        this.math = math;
    }

    public int highest() {
/*        int max = (english > math) ? english : math;
        return max;*/
/*        if (english > math) {
            max = english;
        } else {
            max = math;
        }*/
        return (english > math) ? english : math;
    }

    public void print() {
        System.out.println(name + "\t英文分數" + english + "\t數學分數" +math +
                "\t平均" + (english+math)/2);
    }
}

File: src/main/kotlin/com/cyt/student/Student.kt
package com.cyt.kotlin
import java.util.*

fun main() {
    //userinput()
}

class Student(var name:String?, var english:Int, var math:Int) {
    fun print() {
        println(name + "\t" + english + "\t" + math +"\t" + (english+math)/2)
    }
    fun nameCheck() {
        println(name?.length)
    }
}

private fun userinput() {
    val scanner = Scanner(System.`in`)
    print("Please enter student's name:")
    //var name = scanner.next()
    var name = null
    print("Please enter English grades::")
    var english = scanner.nextInt()
    print("Please enter student's grades:")
    var math = scanner.nextInt()
    val stu = Student(name, english, math)
    stu.print()
    stu.nameCheck()
}

